charts/templates/tokendings.yaml

```diff
@@ -59,6 +59,9 @@ spec:
         databases:
           - name: tokendings
             envVarPrefix: DB
+        flags:
+          - name: max_connections
+            value: "200"
```

Comment:
This might cause a restart of the sql instance. Otherwise looks good to me!

Reply:
Yes, it seems so. I suggest merging this during maintenance hours when traffic is low.